achievements.lua

```diff
@@ -19,13 +19,13 @@
 			--   instead of using metadata.bundleID so that it won't get 
 			--   mangled by online sideloading. Plus, this way multi-pdx
 			--   games or demos can share achievements.
-			gameID = "com.yourcompany.yourgame",
+			gameID = "com.example.yourgame",
```

Comment:
I know I'm really being pedantic here, but: I changed this because `example` is the canonical one specifically set aside for that purpose by the internet powers that be. See www.example.com -- this is guaranteed to _always_ be an example, (I think even with `.net`, `.org` and such other TLDs) unlike 'yourcompany.com'.

I'm not against changing it back per-se, of course, like I said, it's a pedantic reason, and like you said, the 'yourcompany' makes it maybe a bit more clear to people what sort of thing is expected there.